### ISSUE TITLE
Set default aria-label for breadcrumb

### ISSUE THREAD

**Issue by kush-savani** [IBM/carbon-components-angular#2185]
## Detailed description

In `ibm-breadcrumb` component, breadcrumb `nav` tag and overflow menu should have default `aria-label` value same as mention in [carbon design](https://carbondesignsystem.com/components/breadcrumb/accessibility/)

This is  a bug?

Related to `<ibm-breadcrumb>`

Codesandbox - [ibm-breadcumb](https://angular.carbondesignsystem.com/?path=/story/components-breadcrumb--model)

## Additional information

FYI :- Carbon React uses 'Breadcrumb' by default [ref](https://github.com/carbon-design-system/carbon/blob/main/packages/react/src/components/Breadcrumb/Breadcrumb.js#L32)
